build.gradle

```diff
@@ -310,6 +311,14 @@ task testIntegration(type: Test) {
     }
 }
 
+task testUnit(type: Test) {
+    useJUnit {
+        includeCategories 'bio.terra.category.Unit'
+        excludeCategories 'bio.terra.category.Connected', 'bio.terra.category.StairwayUnit',
```

Comment:
The examples in the documentation always list all of the excludes, but I am not sure it is needed. The unit tests weren't running. Maybe we should remove the excludes?

Reply:
I was wondering this earlier -- will remove and see how it goes.